runtime/master/src/main/java/org/apache/nemo/runtime/master/metric/MetricStore.java

```diff
@@ -245,6 +246,8 @@ public void dumpAllMetricToFile(final String filePath) {
     try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
       final String jsonDump = dumpAllMetricToJson();
       writer.write(jsonDump);
+    } catch (final FileNotFoundException e) {
+      LOG.warn("fail to write metric to local file: {}", e);
```

Comment:
fail -> Failed

Reply:
or maybe "Failure while writing metric ..."

Reply:
i changed error message to Failure while writing metric ...